app/views/idv/in_person/state_id.html.erb

```diff
@@ -128,6 +128,7 @@
       <% [
            [:default, state_id_number_hint_default],
            ['TX', t('in_person_proofing.form.state_id.state_id_number_texas_hint')],
+           ['FL', t('in_person_proofing.form.state_id.state_id_number_florida_hint')],
```

Comment:
WDYT of keeping everything after the default value alphabetized?
```suggestion
           ['FL', t('in_person_proofing.form.state_id.state_id_number_florida_hint')],
           ['TX', t('in_person_proofing.form.state_id.state_id_number_texas_hint')],
```

Reply:
that makes sense! i'll update